CHANGELOG/CHANGELOG-1.5.md

```diff
@@ -1,5 +1,16 @@
 # Changelog 1.5
 
+## 1.5.3 - 2022-05-12
+
+### Fixed
+
+- Clean up stale routes installed by AntreaProxy when ProxyAll is disabled. ([#3465](https://github.com/antrea-io/antrea/pull/3465), [@hongliangl])
```

Comment:
@xliuxu I just realized this is not backported when using `./hack/release/prepare-changelog.sh --release 1.5.3` to get authors of this release.

Reply:
Sorry for missing this. I did not run this script as I assume it will not work for patch releases.